### DISCUSSION TITLE
How to programmatically remove LookupEditor attribute from a dialog field?

### DISCUSSION THREAD

**Discussion by ibrahimGit123** [serenity-is/Serenity#6718] [Help] [answered]
I have the LookupEditor attribute on a form field. I would like to remove this editor and convert the field to a plain string after dialog load (e.g. if the field already has a value).

**Comment by wezmag** [comment, marked as answer]
You can have another plain text field in the same form and show/hide it programmatically.
It will be much easier than changing a LookupEditor to a TextEditor.

**Reply by ibrahimGit123** [reply]
Right, that can work

**Reply by LSNicholls** [reply]
Would it also work, in your case, to just set readonly, maybe with a CSS class if you don't like the look?  That's what I have been doing, in general, for this type of thing.  It seems like it takes fewer resources than adding a second field?

At one point I thought I saw an IF attribute, and tried to figure out how that could be used for stuff like this, but either I was dreaming or it was an intellisense glitch ;-).
